style.css

```diff
@@ -0,0 +1,219 @@
+
+body{
+  color:white;
+  margin:0px;
+  font-family: 'Baloo Tamma 2', cursive;
+  background-image: url('./assets/texture_08_by_oleo_kun_d1a4a8l.jpg');
+}
+html { font-size: 13px; 
+}
+/** Header **/
+header{
+	padding: 30px;
+	background-color: whitesmoke;
+	height: 70px;
+	display: flex;
+	align-items: center;
+}
+header h1{
+	font-size: 40px;
+	margin:0px 30px 0px 10px;
+	padding:0px;
+	color:black;
+}
+
+header a{
+	text-decoration: none;
+	color: black;
+	font-size: 16px;
+}
+nav{
+	display: none;
+}
+a:hover {
+	background-color:white;
+  }
+.icon .nav_icon {
+	height: 60px;
+	width: 60px;
+  }
+.navbar_tablet{
+	display:none; 
+ }
+ header h2{
+	display:none;
+}
+/*** Big news section **/
+ .big-news{
+	max-width: 1200px;
+	margin: 0 auto;
+	display:flex;
+	flex-direction: column;
+	justify-content: center;
+	padding-top: 30px;
+ }
+ .news{
+	padding:3rem;
+  	width:480px;
+  	height:90%;
+
+  }
+  .news img { 
+	width:400px;
+  height:90%
+} 
+/*** News cards section **/
+.card {
+  padding: 3rem;
+  width:300px;
+  height:90%; 
+}
+
+.news_cards {
+	max-width: 1000px;
+	margin: 0 auto;
+	display: grid;
+	grid-gap: 3rem;
```

Comment:
Great job with grid approach, but your website just after entering desktop sizes has horizontal scroll - row of cards is wider than the browser window itself